metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -46,38 +46,59 @@ def kubeflow_pipelines(obj):
     "--experiment-name",
     "experiment_name",
     default=DEFAULT_EXPERIMENT_NAME,
-    help="the associated experiment name for the run",
+    help="The associated experiment name for the run",
 )
 @click.option(
     "--run-name",
     "run_name",
     default=DEFAULT_RUN_NAME,
-    help="name assigned to the new KFP run",
+    help="Name assigned to the new KFP run",
 )
 @click.option(
     "--namespace",
     "namespace",
     default=KFP_SDK_NAMESPACE,
-    help="namespace of your run in KFP.",
+    help="Namespace of your run in KFP.",
 )
 @click.option(
     "--api-namespace",
     "api_namespace",
     default=KFP_SDK_API_NAMESPACE,
-    help="namespace where the API service is run.",
+    help="Namespace where the API service is run.",
 )
 @click.option(
     "--yaml-only",
     "yaml_only",
     is_flag=True,
     default=False,
     help="Generate the KFP YAML which is used to run the workflow on Kubeflow Pipelines.",
+    show_default=True,
 )
 @click.option(
     "--pipeline-path",
     "pipeline_path",
     default=DEFAULT_KFP_YAML_OUTPUT_PATH,
-    help="the output path (or filename) of the generated KFP pipeline yaml file",
+    help="The output path of the generated KFP pipeline yaml file",
+)
+@click.option(
+    "--s3-code-package/--no-s3-code-package",
```

Comment:
Why do you also `--no-s3-code-package` here? There should be just one flag, `s3-code-package`, which indicates whether or not the code package is saved in s3.

Reply:
This is the click way to enable or disable a flag type argument. 
see https://click.palletsprojects.com/en/7.x/options/#boolean-flags

```python
import sys

@click.command()
@click.option('--shout/--no-shout', default=False)
def info(shout):
    rv = sys.platform
    if shout:
        rv = rv.upper() + '!!!!111'
    click.echo(rv)
```